mypy/build.py

```diff
@@ -275,9 +275,23 @@ def compute_search_paths(sources: List[BuildSource],
     if alt_lib_path:
         mypypath.insert(0, alt_lib_path)
 
+    package_path = tuple(_get_site_packages_dirs(options.python_executable))
+    for site in package_path:
+        if site in mypypath:
+            print("site-packages is in the MYPYPATH. Please remove it.", file=sys.stderr)
+        elif site in python_path:
+            print("site-packages is in the PYTHONPAT. Please change directory so it is not.",
+                  file=sys.stderr)
+        elif site in lib_path:
+            print("this should never, ever happen", file=sys.stderr)
```

Comment:
This needs a capital letter and a trailing period. Also why should this never happen? And if if *can't* happen perhaps it should be an assert?

Reply:
It should never happen because the path to the typeshed stubs should never include a site-package directory. I will change it to an assert.